src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx

```diff
@@ -77,13 +76,13 @@ You can filter and group data by attributes in various categories; for example:
 To see the list of available attributes, click the funnel <Icon name="fe-filter"/> icon.
 
 <img
-  title="Filterable geography UI"
-  alt="Screenshot of Filterable geography UI in New Relic"
+  title="Geography UI"
+  alt="Screenshot of Geography UI in New Relic"
   src={browserGeoKeyAttributes}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
```

Comment:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) > **Geography > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
```

Reply:
IIRC for actions not based in the UI, we break the bold. but i might x2 check in the style guide